docroot/modules/custom/user_mail/user_mail.services.yml

```diff
@@ -0,0 +1,5 @@
+services:
+  logger.channel.user_mail:
+    class: Drupal\Core\Logger\LoggerChannel
+    factory: logger.factory:get
+    arguments: ['user_mail']
```

Comment:
where are we using this?

Reply:
in WelcomeMail.php:83. 
https://github.com/amansrivastava/d9-dev/pull/28/files#diff-a5e1492db63f9eec8c3c9d419b31b86a8c683f26a891bca1328f185e27a34276R83